src/algebra/homology/image_to_kernel.lean

```diff
@@ -202,6 +207,9 @@ lemma image_subobject_map_comp_image_to_kernel (p : α.right = β.left) :
 by { ext, simp [p], }
 
 variables [has_cokernel (image_to_kernel f g w)] [has_cokernel (image_to_kernel f' g' w')]
+variables [has_cokernel (image_to_kernel f₁ g₁ w₁)]
+variables [has_cokernel (image_to_kernel f₂ g₂ w₂)]
+variables [has_cokernel (image_to_kernel f₃ g₃ w₃)]
```

Comment:
```suggestion
variables [has_cokernel (image_to_kernel f g w)] [has_cokernel (image_to_kernel f' g' w')]
  [has_cokernel (image_to_kernel f₁ g₁ w₁)]
  [has_cokernel (image_to_kernel f₂ g₂ w₂)]
  [has_cokernel (image_to_kernel f₃ g₃ w₃)]
```

Reply:
I think it's quite common to repeat `variables` on each line. Just like with imports.